lang/en.json

```diff
@@ -496,6 +496,9 @@
     "Sense": "Sense",
     "the Terrain": "the Terrain",
     "chosen Lore": "chosen Lore",
+    "Charging": "Charging",
+    "Players": "Player",
```

Comment:
Should be "Players"?

Reply:
@Yasnen Can you fix this? I can then merge